library/src/schemas/special/special.test.ts

```diff
@@ -5,7 +5,7 @@ import { special } from './special.ts';
 
 type PixelString = `${number}px`;
 const isPixelString = (input: unknown) =>
-  typeof input === 'string' && /^\d+px$/.test(input);
+  typeof input === 'string' && /^\d+px$/u.test(input);
```

Comment:
Do you know why it is recommended to at the unicode flag?

Reply:
https://mathiasbynens.be/notes/es6-unicode-regex#:~:text=Recommendations%20for%20developers
so when we use `/ui`, we have to be careful in regexps where `\b` or `\W` is in (there is no such regexp in the code base)

Basically, `/u` turns on strict mode for regexps https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/unicode#:~:text=There%20are%20other%20changes%20to%20the%20parsing